lib_tflite_micro/src/tflite-xcore-kernels/xcore_ops.cc

```diff
@@ -29,6 +29,8 @@ void RegisterXCOps(MicroOpResolver *res) {
                       tflite::ops::micro::xcore::Register_XC_lookup());
   resolver->AddCustom(tflite::ops::micro::xcore::XC_pad_OpCode,
                       tflite::ops::micro::xcore::Register_XC_pad());
+  resolver->AddCustom(tflite::ops::micro::xcore::XC_pad_OpCode,
```

Comment:
I think the opcode should also be `XC_pad_3_to_4`

Reply:
fixed